lib/plugins/inspec-parallel/lib/inspec-parallel/super_reporter/base.rb

```diff
@@ -2,6 +2,8 @@ module InspecPlugins::Parallelism
   class SuperReporter
 
     def self.make(type, job_count, invocations)
+      puts "Press CTL+C to stop\n"
```

Comment:
Ah, we can't output anything here. We may have other super-reporters that are intentionally silent.

Reply:
Got it!